lib/puppet/application/device.rb

```diff
@@ -133,6 +137,10 @@ def help
   appending nature of logging. It must be appended manually to make the content
   valid JSON.
 
+* --user:
+  The user to run as. Due to a bug (PUP-1391), '--user=root' is required, 
```

Comment:
We shouldn't be referencing tickets here. Just omit the entire beginning of that sentence, "Due to a bug (PUP-1391)"

Reply:
I've made that change and squashed the subsequent commit.

I just remembered that I have a PR for PUP-1391: 

https://github.com/puppetlabs/puppet/pull/6026

If 6026 is merged and released, that sentence will become unnecessary.